server/src/main/java/org/elasticsearch/cluster/SnapshotsInProgress.java

```diff
@@ -91,12 +91,13 @@ public String toString() {
         private final ImmutableOpenMap<String, List<ShardId>> waitingIndices;
         private final long startTime;
         private final long repositoryStateId;
+        private final boolean newFormat;
```

Comment:
This might not age well without a comment and/or an explicit name. Maybe something like `useShardGenerations`?

Reply:
jup :) that sounds better